frontend_tests/puppeteer_tests/06-edit.ts

```diff
@@ -14,8 +14,12 @@ async function trigger_edit_last_message(page: Page): Promise<void> {
 async function edit_stream_message(page: Page, topic: string, content: string): Promise<void> {
     await trigger_edit_last_message(page);
 
-    await page.evaluate(() => $(".message_edit_topic").val(""));
-    await page.evaluate(() => $(".message_edit_content").val(""));
+    await page.$eval(".message_edit_topic", (el) => {
+        (el as HTMLInputElement).value = "";
+    });
+    await page.$eval(".message_edit_content", (el) => {
+        (el as HTMLTextAreaElement).value = "";
+    });
```

Comment:
Same with these -- should this just be some sort of `page.type` type command to clear the text boxes?

Reply:
I couldn't find something similar like that. 
I found this related [issue](https://github.com/puppeteer/puppeteer/issues/3094) but it was rejected by one of a maintainer of the project [here](https://github.com/puppeteer/puppeteer/issues/3094#issuecomment-581722198).

One alternative to this will be, 
```javascript
// This will select all the text.
await page.click("selector", {clickCount: 3});
await page.type("selector", "new text");
```

WDYT?

Reply:
Yeah, that sounds better to me.  In general we want to do as little `page.$eval` as we can that could be better expressed via the builtins.